tests/Unit/Elliptic/Actions/Test_InitializeSystem.cpp

```diff
@@ -16,6 +16,7 @@
 #include "Domain/CoordinateMaps/Affine.hpp"
 #include "Domain/CoordinateMaps/CoordinateMap.hpp"
 #include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.tpp"
```

Comment:
Is this one necessary? The only place the `ProductOf...Maps` appears is in `PUPable_reg`, and I don't know whether that pattern constructs the object.